lib/rubocop/cop/layout/tab.rb

```diff
@@ -6,6 +6,16 @@ module RuboCop
   module Cop
     module Layout
       # This cop checks for tabs inside the source code.
+      #
+      # @example
+      #   # This example uses a tab for indentation.
+      #   # bad
+      #     x = 0
```

Comment:
It's a bit hard to see anything on this example 😄  Maybe we should show the tab and spaces in some more visual manner?

Reply:
I agree, but I'm not sure what that would be. Particularly something that would be valid ruby syntax. These appear to be valid, but perhaps they'd cause confusion? Open to any ideas.

```ruby
__x = 10
..x = 10
```

Reply:
Or you can just say "Imagine a tab here" "Imagine 2 spaces". I'd also use some nested code (e.g. `def`) for this, as for a single line it's hard to see the leading indent at all.